### DISCUSSION TITLE
No Audio in Home Assistant or Scrypted

### DISCUSSION THREAD

**Discussion by ikolsteren** [gtxaspec/wz_mini_hacks#488] [Q&A]
I have it all working but can't get the audio to come through to either Home Assistant or scrypted
I only gave the one stream and am using the URL rtsp://admin:1234@192.168.50.132:8554/unicast for generic camera or for a direct WebRTC stream
The audio using the same URL works in VLC
Is there a switch I need to add to specify where the audio stream is?

Update: Got Scrypted to work with audio - just can't get the audio to come into Home Assistant.   For Scrypted I had to do some transcoding (I think thats what fixed it).  Did lots of changes and it works in there now

**Comment by KyleStilkey** [comment]
Home assistant doesn't support audio due to cosec limitation. I'm not 100% sure but I believe you might be able to get it to work if router through RTSP.
